cognite_toolkit/cdf_tk/templates/_templates.py

```diff
@@ -342,6 +349,38 @@ def process_function_directory(
             exit(1)
 
 
+def process_files_directory(
+    files: list[Path],
+    yaml_dest_path: Path,
+    module_dir: Path,
+    build_dir: Path,
+    verbose: bool = False,
+) -> None:
+    if len(files) == 0:
+        return
+    try:
+        file_def = FileMetadataList.load(yaml.safe_load(yaml_dest_path.read_text()))
```

Comment:
The `.load` method has a built in yaml loader, so you don't have to do it :) 
```suggestion
        file_def = FileMetadataList.load(yaml_dest_path.read_text())
```